qiskit_ibm_runtime/accounts/account.py

```diff
@@ -77,10 +77,8 @@ def to_saved_format(self) -> dict:
     def from_saved_format(cls, data: dict) -> "Account":
         """Creates an account instance from data saved on disk."""
         proxies = data.get("proxies")
-        auth = data.get("auth")
-        channel = "ibm_quantum" if auth == "legacy" else "ibm_cloud"
         return cls(
-            channel=data.get("channel", channel),
```

Comment:
Given is is guaranteed that `from_saved_format` is only called after `migrate`, we don't need this code.

It was hiding a bug in the migration code and prevented the test cases from failing.